app/build.gradle

```diff
@@ -17,6 +17,8 @@ import org.mozilla.fenix.gradle.tasks.LintUnitTestRunner
 
 import static org.gradle.api.tasks.testing.TestResult.ResultType
 
+def useFreeImplementation = project.properties['org.mozilla.fenix.useFreeImplementation'].toBoolean()
```

Comment:
This might be better off as another build variant, like debug.

Reply:
So this is done. I've used build flavors which definitely makes more sense. Needing help to update taskcluster.